src/HTSPConnection.cpp

```diff
@@ -407,39 +408,51 @@ bool CHTSPConnection::SendHello ( void )
     return false;
   
   /* Process */
+  const char *serverName;
+  const char *serverVersion;
+  uint32_t htspVersion;
   const char *webroot;
   const void *chal;
   size_t chal_len;
-  htsmsg_t *cap;
 
   /* Basic Info */
   webroot = htsmsg_get_str(msg, "webroot");
-  m_serverName    = htsmsg_get_str(msg, "servername");
-  m_serverVersion = htsmsg_get_str(msg, "serverversion");
-  m_htspVersion   = htsmsg_get_u32_or_default(msg, "htspversion", 0);
-  m_webRoot       = webroot ? webroot : "";
+  serverName = htsmsg_get_str(msg, "servername");
+  serverVersion = htsmsg_get_str(msg, "serverversion");
+  htspVersion = htsmsg_get_u32_or_default(msg, "htspversion", 0);
+  webroot = webroot ? webroot : "";
```

Comment:
Note to self that the ternary here should be removed since we check for NULL before storing, and `ServerInformation::m_webRoot` defaults to an empty string anyway.